bin/c8.js

```diff
@@ -40,3 +29,20 @@ function outputReport () {
     omitRelative: argv.omitRelative
   })
 }
+
+(async function run () {
+  if (argv._[0] === 'report') {
+    argv = yargs.parse(process.argv) // support flag arguments after "report".
+    outputReport()
+  } else {
+    if (argv.clean) {
+      await _p(rimraf)(argv.tempDirectory)
+      await _p(fs.mkdir)(argv.tempDirectory, { recursive: true })
```

Comment:
unfortunately this functionality will not yet work: https://github.com/nodejs/node/pull/21875